assets/js/langchoose.js

```diff
@@ -84,6 +84,45 @@ function selectLanguage(lang) {
     });
 }
 
+// copyNextCodeBlock copies the contents of the currently active code block following the element,
+// elem, which is the "copy" icon being clicked.
+function copyNextCodeBlock(elem) {
```

Comment:
I was going to be a bit smarter about copying `language-bash` and `language-shell`, stripping `"$ "` at the beginning of any lines.

I could also see needing to have a way to opt-out or specify which lines should be copyable (in the case of `language-bash` that has an initial command, and then the output subsequently, where we'd just want to copy the command.).